pymc/variational/opvi.py

```diff
@@ -76,6 +72,10 @@ class VariationalInferenceError(Exception):
     """Exception for VI specific cases"""
 
 
+class NotImplementedInference(VariationalInferenceError, NotImplementedError):
+    """Marking non functional parts of code"""
```

Comment:
Can you elaborate on this docstring?
In which situations should we use this type instead of either of the base types?